### PR TITLE
1191 npm outdated

### PR DESCRIPTION
**Pull request by DutchBen**

### CODE REVIEW

**Review by Georgi2704 — APPROVED** [review]

File: packages/orchestrator-ui-components/src/components/WfoWorkflowSteps/WfoStepList/WfoStepList.tsx [hunk 1]
```diff
@@ -69,10 +69,13 @@ export const WfoStepList = React.forwardRef(
         }));
 
         const getReferenceCallbackForStepId =
-            (stepId: string) => (node: HTMLDivElement | null) =>
-                node
-                    ? stepReferences.current.set(stepId, node)
-                    : stepReferences.current.delete(stepId);
+            (stepId: string) => (node: HTMLDivElement | null) => {
```
Comment (ricardovdheijden) [inline packages/orchestrator-ui-components/src/components/WfoWorkflowSteps/WfoStepList/WfoStepList.tsx:72]: This does not seem related to updating package versions. What are you trying to fix here?

Comment (DutchBen) [reply packages/orchestrator-ui-components/src/components/WfoWorkflowSteps/WfoStepList/WfoStepList.tsx:72]: It's related to the typescript version upgrade. This fixes the error it throws after the upgrade.

**Review by ricardovdheijden — APPROVED** [review]
